scripts/travis/gtclang-driver-install.sh

```diff
@@ -32,15 +32,15 @@ cd bundle
 mkdir build
 cd build
 
-export PYTHON_DIR=/opt/python/3.5.3
+export PYTHONPATH=`which python3`
```

Comment:
I went with 3.5 but I guess this is not needed.

Also for sake of consistency I've moved python to be a package in .travis.yml:

```
language: c++
addons:
  apt:
    sources:
      - ubuntu-toolchain-r-test
      - deadsnakes
    packages:
      - python3.5

      - g++-5
...
```